src/govuk/components/character-count/character-count.mjs

```diff
@@ -120,6 +120,13 @@ CharacterCount.prototype.init = function () {
   var $textarea = this.$textarea
   var $fallbackLimitMessage = document.getElementById($textarea.id + '-info')
 
+  // Inject a decription for the textarea if none is present already
+  // for when the component was rendered with no maxlength, maxwords
+  // nor custom fallbackHintText
+  if ($fallbackLimitMessage.textContent.match(/^\s*$/)) {
+    $fallbackLimitMessage.textContent = this.i18n.t('fallbackHint', { count: this.maxLength })
```

Comment:
If folks aren't using the Nunjucks macro, and don't have the corresponding changes to pass us the fallback hint, this could result in an error being thrown.

Given we're already in a 'best efforts' context trying to do something sensible, I think we should avoid that by guarding this – either by checking for `fallbackHint` being set or just wrapping this in a try catch?

Reply:
I don't think anything will be thrown there:
- `$fallbackLimitMessage` was already used beforehand, it's the reference for where to insert the visible and screen-reader containers
- the I18n class will not throw if there's no entry for `fallbackHint` in its content, it [returns the lookup key if no translation was provided](https://github.com/alphagov/govuk-frontend/blob/main/src/govuk/i18n.mjs#L43-L61). 

This would set a default of `fallbackHint` as a description for assistive technologies. How about we provide our own default of `{other: ''}` to ensure we have a smooth experience on that side? That'll set an empty string if no alternative is provided by data attributes (or Nunjucks) and warn people in the console if they needed to use any of the other keys.

Thinking about it, we should probably warn in I18n if no key is found for the translation as well.

Reply:
Because we're treating `fallbackHint` as a pluralised string, we'll call `getPluralSuffix` before we get to that code, and  `getPluralSuffix` will throw if `fallbackHint.other` is not defined.

https://github.com/alphagov/govuk-frontend/blob/5e0746848368760a1a263057b4957369e922a82d/src/govuk/i18n.mjs#L162-L179

> How about we provide our own default of {other: ''} to ensure we have a smooth experience on that side?

That could work 👍🏻